src/components/books/bookList.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './book.css';
+
+function BookList({ List: { title, author } }) {
+  return (
+    <div className="bookli">
+      <div className="main">
+        <h2>{title}</h2>
+        <p className="bookAuthor">{author}</p>
+      </div>
+      <div className="list">
+        <a href="/comment" className="comment">
+          Comments
+        </a>
+
+        <a href="/remove" className="comment">
+          Remove
+        </a>
```

Comment:
As per the project requirements, this should be a button. Kindly change it to a button instead of a link. 
![image](https://user-images.githubusercontent.com/33268927/221213436-8f0b5c7b-3982-406e-8e3e-dc10b05f722f.png)